cluster/scripts/lodestar/keystore-import.sh

```diff
@@ -5,29 +5,45 @@ INFO="[ INFO | lodestar-keystore-import ]"
 function import_keystores_to_lodestar() {
 
     local charon_keys_dir=${CHARON_ROOT_DIR}/validator_keys
-    local lodestar_keys_dir=${VALIDATOR_DATA_DIR}/keystores
 
-    for f in "${charon_keys_dir}"/keystore-*.json; do
+    local validator_keys_dir=${VALIDATOR_DATA_DIR}/keystores
```

Comment:
These are the default dirs and will likley never change but if you wanna be more explicit can pass those flags to lodestar vc to make sure it's the same location